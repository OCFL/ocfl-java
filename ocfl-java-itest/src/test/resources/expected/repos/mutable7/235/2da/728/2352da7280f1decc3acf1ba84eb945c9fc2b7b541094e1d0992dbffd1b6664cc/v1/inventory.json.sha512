85275aae85e25ad0aa59ec182e7e0ab4675adc0dd3fe36e3487af1d3d91f54700cd54e5b753fac22e9bac45701bc22076dd99c3218dc265ea75e9b6a0cf0716d  inventory.json
