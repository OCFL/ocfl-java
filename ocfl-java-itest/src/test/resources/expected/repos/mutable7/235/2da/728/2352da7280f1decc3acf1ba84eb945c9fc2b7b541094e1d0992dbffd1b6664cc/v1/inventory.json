{
  "id" : "o1",
  "type" : "https://ocfl.io/1.0/spec/#inventory",
  "digestAlgorithm" : "sha512",
  "head" : "v1",
  "contentDirectory" : "content",
  "fixity" : { },
  "manifest" : {
    "b81b77214f30e9f9dca7f19800a9d8c8683d2e40842dbad3027bcd828e39a19fd085b02c22f704fa865619755bbcc47fcd4de236fbbe6fd0a9451c75d562c943" : [ "v1/content/file1.txt" ]
  },
  "versions" : {
    "v1" : {
      "created" : "2019-08-05T15:57:53Z",
      "message" : "commit message",
      "user" : {
        "name" : "Peter",
        "address" : "peter@example.com"
      },
      "state" : {
        "b81b77214f30e9f9dca7f19800a9d8c8683d2e40842dbad3027bcd828e39a19fd085b02c22f704fa865619755bbcc47fcd4de236fbbe6fd0a9451c75d562c943" : [ "file1.txt" ]
      }
    }
  }
}